node/src/GlideClient.ts

```diff
@@ -208,6 +207,10 @@ export class GlideClient extends BaseClient {
      *
      * @see {@link https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#custom-command|Valkey Glide Wiki} for details on the restrictions and limitations of the custom command API.
      *
```

Comment:
Added documentation here. Not sure, why was this not added. If not required, can remove this.

Reply:
customCommand is the first command... so maybe it was just a mistake...?